.ci/travis/travis.sh

```diff
@@ -2,6 +2,12 @@
 # Attention, there is no "-x" to avoid problems on Travis
 set -e
 
+removeFolderWithProtectedFiles() {
+  find . -delete
```

Comment:
please do `cd "$1"` to do deletion inside target folder.

Reply:
I'm curious; why `cd ..` then `cd "$1"`?  To make sure we're in the right directory?

Reply:
yes, and you raise a good point we should do `cd -`  at the end to return back to where we were before function execution.

Reply:
Please send separate PR to add `cd -`.
I am merging this PR sooner as another PR depends on it.